queries/dart/highlights.scm

```diff
@@ -12,7 +12,12 @@
 (marker_annotation
   name: (identifier) @annotation)
 
-; Operators
+; Operators and Tokens
+
+; FIXME: currently this errors
+; (template_substitution
+;   "${" @punctuation.special
+;   "}" @punctuation.special) @embedded
```

Comment:
What is the error here ?

Reply:
The nodes are still hidden in the grammar. There are a lot of hidden nodes upstream I've been making some PRs to reveal some of them. I think there's more issues with `$` though as the ranges in the tree are incorrect anyway so think this one will be a bigger issue in the grammar than just removing the underscore